department-of-veterans-affairs/abd-vro
# Add additional Contention Classification database table columns

## What was the problem?
We need to store additional data that were not being captured before.

Associated tickets or Slack threads:
- #2085 

## How does this fix it?[^1]

- added migration script to add new columns that are defined in #2085
- added a new Util class and its method to populate `BieMessagePayload` object using java reflection 
- added unit test for the util class

## How to test this PR
- run local integration and unit test on `svc-bie-kafka` service


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
